compose/docker-compose.yml

```diff
@@ -25,17 +25,17 @@ x-op-app: &app
     # set to true to enable the email receiving feature. See ./docker/cron for more options
     IMAP_ENABLED: "${IMAP_ENABLED:-false}"
   volumes:
-    - "opdata:/var/openproject/assets"
+    - "${OPDATA:-opdata}:/var/openproject/assets"
 
 services:
   db:
     image: postgres:13
     <<: *restart_policy
     stop_grace_period: "3s"
     volumes:
-      - "pgdata:/var/lib/postgresql/data"
+      - "${PGDATA:-pgdata}:/var/lib/postgresql/data"
     environment:
-      POSTGRES_PASSWORD: p4ssw0rd
+      POSTGRES_PASSWORD: ${POSTGRES_PSWD:-p4ssw0rd}
```

Comment:
I would suggest naming the environment variable `POSTGRES_PASSWORD` as well to be consistent with what's used in the container.

Reply:
No problem :)